# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/webrtc/libwebrtcglue/AudioConduit.cpp
# Commit: da26bac32c32
# Full Hash: da26bac32c327cd814c4b13630061f7a5c07c339
# Author: Andreas Pehrson <apehrson@mozilla.com>
# Date: 2021-11-01 18:17:33
# Regressor Bug: 1654112
# File Overlap Count: 3
# Description:
#   Bug 1654112 - Clean up and sort Audio/Video/MediaConduit includes. r=ng
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D102289
# ==============================================================================

diff -r ceed49af0fad -r da26bac32c32 dom/media/webrtc/libwebrtcglue/AudioConduit.cpp
--- a/dom/media/webrtc/libwebrtcglue/AudioConduit.cpp	Mon Jan 18 17:32:49 2021 +0100
+++ b/dom/media/webrtc/libwebrtcglue/AudioConduit.cpp	Tue Jan 19 10:59:28 2021 +0100
@@ -2,34 +2,25 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this file,
  * You can obtain one at http://mozilla.org/MPL/2.0/. */
 
+#include "AudioConduit.h"
+
 #include "common/browser_logging/CSFLog.h"
-#include "nspr.h"
+#include "mozilla/dom/RTCRtpSourcesBinding.h"
+#include "mozilla/media/MediaUtils.h"
+#include "mozilla/Telemetry.h"
+#include "transport/runnable_utils.h"
 
+// libwebrtc includes
+#include "api/audio_codecs/builtin_audio_encoder_factory.h"
+#include "audio/audio_receive_stream.h"
+
+// for ntohs
 #ifdef HAVE_NETINET_IN_H
 #  include <netinet/in.h>
 #elif defined XP_WIN
 #  include <winsock2.h>
 #endif
 
-#include "AudioConduit.h"
-#include "nsCOMPtr.h"
-#include "mozilla/dom/RTCRtpSourcesBinding.h"
-#include "mozilla/media/MediaUtils.h"
-#include "nsServiceManagerUtils.h"
-#include "nsThreadUtils.h"
-#include "mozilla/Telemetry.h"
-#include "transport/runnable_utils.h"
-
-#include "pk11pub.h"
-
-#include "api/audio_codecs/builtin_audio_decoder_factory.h"
-#include "api/audio_codecs/builtin_audio_encoder_factory.h"
-#include "audio/audio_receive_stream.h"
-#include "modules/audio_processing/include/audio_processing.h"
-#include "modules/rtp_rtcp/include/rtp_rtcp.h"
-#include "modules/rtp_rtcp/source/rtp_packet_received.h"
-#include "system_wrappers/include/clock.h"
-
 #ifdef MOZ_WIDGET_ANDROID
 #  include "AndroidBridge.h"
 #endif